src/services/prisma-datastore/db-init.ts

```diff
@@ -0,0 +1,44 @@
+import { PrismaClient, Project, Gateway, DeviceReadingSchema, Reading, Prisma, Node } from '@prisma/client'
+
+import Config from '../../../config'
```

Comment:
It'd be better to have the Config depend on Prisma, not the other way around. Otherwise this depends on Config which depends on everything and then everything's dependent on everything. The config is the lowest level implementation detail in other words.

For Example, imagine in the future if we have some sort of 'context' defining the project, company name, etc. it would suck to have to rewrite this component.

Reply:
I see what you mean, but not how it plays out. Presently, Config just depends on the `process.env`, which is how we are abstracting the external environment. Perhaps it should be renamed `Environment` and config is something else that we don't yet have (probably best embodied by the bits of code in ServiceLocator that builds the AppService. )